Makefile

```diff
@@ -1,3 +1,3 @@
 all: src/*.cpp inc/*.hpp
-	g++ -std=c++17 src/*.cpp -o main
+	g++ -std=c++17 src/*.cpp -Werror -o main
```

Comment:
You should also add another flags like -Wall -Wextra -pedantic.
They will cause a compilation error.